libs/SalesforceSDK/src/com/salesforce/androidsdk/rest/RestClient.java

```diff
@@ -215,60 +234,12 @@ public RequestQueue getRequestQueue() {
 	 * @throws IOException 
 	 */
 	public RestResponse sendSync(RestRequest restRequest) throws IOException {
```

Comment:
Sync and async network requests are done the same way - using OkHttp.
With volley, we used to handle them differently.